extension-manifest-v2/src/background.js

```diff
@@ -756,7 +756,8 @@ function onMessageHandler(request, sender, callback) {
 		return true;
 	}
 	if (name === 'update_database') {
-		updateDBs().then((result) => {
+		updateDBs().then(async (result) => {
+			await common.modules.adblocker.background.adblocker.update();
```

Comment:
Do we need to recover from errors here? The old code also did not try (if updateDBs failed the callback does not fire). Perhaps it is not needed here, so I'm fine with merging as is. Just a comment.